core/src/main/java/org/apache/druid/indexer/TaskStatusPlus.java

```diff
@@ -42,9 +42,12 @@
 
   @Nullable
   private final String errorMsg;
+  @Nullable
+  private final String groupId;
```

Comment:
This sounds an interesting idea, but I'm not sure if it would make sense with Kafka/Kinesis indexing tasks. The kafka/kinesis supervisor spawns indexing tasks but the supervisor is executed using a thread in the overlord. The tasks have the same groupId which is `datasource` name where they ingest data into.

Reply:
Ah, I didn't know there's already a notion of task groups. For consistency, we should continue using "group" then.